### ISSUE TITLE
✅ 🐞 Bug report: padding of the sticky section is changing when Flexbox Container is active

### ISSUE THREAD

**Issue by simkoG** [elementor/elementor#18835]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When I create a sticky to the bottom section within a template and insert it into a page, the section's left and right padding changes when I scroll the page. The correct padding shows only, when the sticky element is non-sticky. 

**The BUG only occurs when the experimental Flexbox Container is active.**

### Steps to reproduce

1. Install Elementor + Elementor Pro (+ Hello theme).
2. Active the Flexbox Container `Elementor > Settings > Experiments > Flexbox Container > Active`. 
3. Create a new template: 
3.a. add a new container
3.b. set the content's max-width to 680px
3.c. create two columns (add some background and border color for reference)
3.d. make the new section sticky (`Advanced > Motion Effects > Sticky: Bottom`)
3.e. save the template
4. Create a new page.
5. Add two dummy paragraph (to enable page scroll).
6. Insert the newly created template between the two paragraphs.
7. Save and preview the newly created page. 

https://user-images.githubusercontent.com/10430702/171810970-0f6eb54d-f8cf-47af-b9c0-7c439a061c2a.mp4

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.52 (Unix) OpenSSL/1.1.1f
	MySQL version: mariadb.org binary distribution v10.5.16
	PHP Version: 8.0.19
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0
	Site URL: https://bisama.us9.instawp.xyz
	Home URL: https://bisama.us9.instawp.xyz
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /index.php/%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.5.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.63 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.6.5
		Author: Elementor.com

	Elementor Pro
		Version: 3.7.1
		Author: Elementor.com

	HelloPack
		Version: 1.1.9
		Author: HelloWP Ltd


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Active by default
	Hide native WordPress widgets from search results: Active by default
	: Inactive by default
	Flexbox Container: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default


== Log ==
	
Log: showing 5 of 52022-06-03 07:00:46 [info] elementor-pro::elementor_pro_updater Started 
2022-06-03 07:00:46 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-06-03 07:00:46 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-06-03 07:00:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.3',
  'to' => '3.7.1',
)]
2022-06-03 07:00:46 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.3',
  'to' => '3.7.1',
)]

JS: showing 1 of 1JS: 2022-06-03 07:31:58 [error X 3][https://bisama.us9.instawp.xyz/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.5:2:391141] Cannot read properties of undefined (reading \'parent\') 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

**Comment by simkoG** [comment]
Template export from the sample video: https://pastebin.com/KUFUd40B

**Comment by nicholaszein** [comment]
Hello @simkoG!

Thank you for your feedback, and for being so patient! 🙏 

We recently released the new **3.16** version of Elementor. And we added a fix for **Sticky Container** padding that also solved your issue.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
